### PR TITLE
Simplify project setup

### PR DESCRIPTION
**Pull request by maciejhein**
To simplify the project setup I've added the following files to repository:
- `google-services.json`
- `keystore.properties`

I also proposes change the content of the wiki page `Getting started the setup` to :
``` 
  1. Clone repository. Make sure to use yarn instead of npm. This project has only yarn.lock lockfile.
  2. Install Android Studio: https://developer.android.com. In Android Studio open PROJECT_ROOT/android directory.
  3. Install IDE to develop React Native App and open ROOT_DIRECTORY with it.
  4. Check out the official React Native (CLI version) tutorial: https://facebook.github.io/reactnative/docs/getting-started and install required dependencies.
  5. Download all npm dependencies using yarn in the root of the project.
  6. Run Virtual Device from AVD Manager in Android Studio.
  7. Run from command line: react-native run-android
```

### CODE REVIEW

**Comment by kwbak** [comment]
Also, we would like to simplify the documentation. Our propositions:

- remove `CONTRIBUTION.md` file - the same content you can find in `README.md`

- remove `GUIDE.md` and:

  - move the Guides section to the wiki page

  - remove Fastlane section - afaik contributors cannot use it because they need firebase private key - can we solve this problem in any different way? If not we should remove this from documentation because now it is useless

**Comment by bklukaczewski** [comment]
👍 It would be good to include also iOS config for Firebase

**Comment by PrzemekPopowicz** [comment]
keystore.properties is empty - is that okay?

**Comment by bklukaczewski** [comment]
It's not really necessary to create virtual device with API level 22. It can be 22 or anything higher.

File: .gitignore [hunk 1]
```diff
@@ -66,14 +66,11 @@ coverage/
 
 # Firebase
 GoogleService-Info.plist
-google-services.json
-firebase-admin.json
```
Comment (bklukaczewski) [inline .gitignore]: admin json should still be in gitignore as it contains sensitive private keys

**Comment by maciejhein** [comment]
> keystore.properties is empty - is that okay?

yes

**Comment by kwbak** [comment]
> 👍 It would be good to include also iOS config for Firebase

@bklukaczewski Where we can find this config?

**Comment by bklukaczewski** [comment]
> > 👍 It would be good to include also iOS config for Firebase
> 
> @bklukaczewski Where we can find this config?

From Firebase web console, same as Android config ;) but if you don't have access to it and you received config file from someone it's okay and I'll add it in separate PR.

**Comment by mgolawski** [comment]
```
  1. Clone repository. Make sure to use yarn instead of npm. This project has only yarn.lock lockfile.
  2. Install Android Studio: https://developer.android.com. In Android Studio open PROJECT_ROOT/android directory. 
  3. Install IDE to develop React Native App and open ROOT_DIRECTORY with it.
  4. Check out the official React Native (CLI version) tutorial: https://facebook.github.io/reactnative/docs/getting-started and install required dependencies.
-- *Given link redirect to the main Open Source facebook page*
-- *Specify the required dependencies. Which node is required? If project is android based do i need XCode? Please add some details to that. I would copy all required steps into wiki, and add the link for deep digging if needed.*
  5. Download all npm dependencies using yarn in the root of the project.
  6. Run Virtual Device from AVD Manager in Android Studio.
--  -- *Specify the supported version of Android*
  7. Run from command line: react-native run-android
```

**Pull request merged**